p9cpu/crates/libp9cpu/src/lib.rs

```diff
@@ -33,3 +35,40 @@ where
         }
     }
 }
+
+pub const NINEP_MOUNT: &str = "mnt9p";
+pub const LOCAL_ROOT_MOUNT: &str = "local";
+
+pub fn parse_namespace(namespace: &str, tmp_mnt: &str) -> Vec<cmd::FsTab> {
```

Comment:
do we need this any more?

Reply:
This is the client side. We still provide cpu_namespace as a convenient way to specify an fstab. The client will convert the namespace to an fstab. The server side does not use namespaces.